pages/linux/fail2ban-client.md

```diff
@@ -0,0 +1,16 @@
+# fail2ban-client
+
+> Configure and control fail2ban server.
+> More information: <https://github.com/fail2ban/fail2ban/>.
+
+- Retrieve current status of the jail service:
+
+`fail2ban-client status {{jail}}`
+
+- Manually unban IP from the jail service:
```

Comment:
I think the `manually` is not needed:
```suggestion
- Unban IP from the jail service:
```
I'm sure @sbrl won't like the `unban` as a verb, maybe he knows a good synonym.

Reply:
I feel like this is probably fine (to use _unban_) as it's the terminology used by Fail2ban. 🤷🏻

Reply:
Unban is fine as a verb here, I think. However, I think we're missing an article. What  about this?
```suggestion
- Unban an IP from the jail service:
```

However, I like @owenvoke's description much better, so I think we should go with that instead.
```suggestion
- Remove the specified IP from the jail service's ban list:
```